src/styles/createTypography.d.ts

```diff
@@ -36,7 +37,7 @@ export interface TypographyStyle {
 
 export type Typography = { [type in Style]: TypographyStyle } & FontStyle;
 
-export type TypographyOptions = Partial<FontStyle> & Partial<Typography>;
+export type TypographyOptions = DeepPartial<TypographyStyle>;
```

Comment:
```diff
-export type TypographyOptions = DeepPartial<TypographyStyle>;
+export type TypographyOptions = DeepPartial<Typography>;
```

Reply:
Why? 
This way you get no mapped type over Style.

Reply:
But `Typography` is already a mapped type over `Style` 🤔